{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/tests/test_drf_views.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from django.test import RequestFactory
+
+from {{ cookiecutter.project_slug }}.users.models import User
+from {{ cookiecutter.project_slug }}.users.api.views import UserViewSet
+from {{ cookiecutter.project_slug }}.users.api.serializers import UserSerializer
```

Comment:
This is causing an isort error in the flake8 checks, should be ordered as below:

```suggestion
from {{ cookiecutter.project_slug }}.users.api.serializers import UserSerializer
from {{ cookiecutter.project_slug }}.users.api.views import UserViewSet
from {{ cookiecutter.project_slug }}.users.models import User
```

Reply:
OK. I mssed isort error.